op-batcher/flags/flags.go

```diff
@@ -110,8 +114,8 @@ var Flags []cli.Flag
 
 func CheckRequired(ctx *cli.Context) error {
 	for _, f := range requiredFlags {
-		if !ctx.GlobalIsSet(f.GetName()) {
-			return fmt.Errorf("flag %s is required", f.GetName())
+		if !ctx.IsSet(f.Names()[0]) {
```

Comment:
Could this out of bounds?

Reply:
There's always at least 1 name in a flag